dbms/src/Storages/Page/V3/CheckpointFile/Proto/manifest_file.proto

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.#pragma once
+
+syntax = "proto3";
+
+package DB.PS.V3.CheckpointProto;
+
+import "common.proto";
+
+/**
+
+A manifest file format V1 consists of the following parts:
+
+- Len|Prefix
+- Len|EditsPart
+- Len|LocksPart
+...
+
+ */
+
+message ManifestFilePrefix {
```

Comment:
do we need to add some fields about the compress algorithm so that we can change the algorithm later?

Reply:
Yes, we can add some settings there when we want to change compression algorithm.